Add automation blueprint for external temperature updates

tbarnekov opened the pull request:
I created an automation blueprint to make it easier to update the external temperature property of the Ally thermostats.

The blueprint supports selecting multiple temperature sensors and multiple thermostats. Temperature sensor values are averaged before being sent to the selected thermostat(s). You'll need to add an automation for every room that has one or more external temperature sensor and one or more thermostats.

The blueprint expects temperature sensor values to be in Celsius. If anyone knows how to support Fahrenheit please let me know.

MTrab commented:
Currently not possible to add to the release as blueprints require a separate release.
But will let this remain open for now

MTrab commented:
Think it would be better to move the YAML file up to `/blueprints` rather than the deep folder structure, as it currently is a manual process to add blueprints.

tbarnekov commented:
Maybe it would be better to post it to the "official" blueprint exchange (https://www.home-assistant.io/get-blueprints) and add a link in the Readme.md instead?

MTrab commented:
Should still not be in that deep of a folder structure. But yeah, might be the best solution.

tbarnekov commented:
I've added a blueprint import button to Usage.md and removed the actual blueprint from the PR.

I opted for a GitHub Gist instead of posting a topic on the Blueprint Exchange for two reasons:

1. The blueprint is only useful if you have the Ally component installed so a link in the documentation is more useful.
2. The blueprint exchange looks like a garbage dump where everyone and their mother dumps their stuff.
